esmvalcore/preprocessor/_regrid.py

```diff
@@ -463,6 +463,26 @@ def regrid(cube, target_grid, scheme, lat_offset=True, lon_offset=True):
     - ``end_latitude``: longitude at the center of the last grid cell.
     - ``step_latitude``: constant latitude distance between grid cell centers.
 
+    To specify a reference dataset with a ``dict``, the following keys are
+    necessary:
+
+    - ``filename``: path to the file that contains the reference grid.
+    - ``project``: project of the reference dataset.
+    - ``dataset``: name of the reference dataset.
+    - ``short_name``: name of the variable in the reference dataset.
+    - ``mip``: name of the MIP table used.
+    - ``frequency``: temporal frequency of the reference dataset.
+    - ``fix_dir``: path to the directory where the new file is ssaved in case
+      fixes are necessary.
+
+    Note
+    ----
+    If ``levels`` is a ``dict`` and it does not contain the key ``filename``,
+    it is automatically assumed that you specified the target grid with
+    ``start_longitude``, ``end_longitude``, etc. If a valid reference dataset
```

Comment:
wait what? A grid is specified by an MxN specification, not by defining a box, am confusado here

Reply:
ah nevermind, it's the `_spec_to_latlonvals()` stuff - can you add a pointer to that func here maybe, that confused me, prob gonna confuse somebody else too